src/index.js

```diff
@@ -79,24 +84,26 @@ module.exports = (options) => {
         proxyUrl,
         payload,
         stream,
+        useBrotli,
     } = opts;
 
     const requestOptions = {
         url,
         method,
-        headers,
+        headers: _.defaults(headers, { 'Accept-Encoding': `gzip, deflate${useBrotli ? ', br' : ''}` }),
         followRedirect,
         maxRedirects,
         timeout: timeoutSecs * 1000,
         rejectUnauthorized: !ignoreSslErrors,
         body: payload,
         json,
-        decodeBody: true,
-        throwHttpErrors,
+        throwHttpErrors: false,
+        stream: true,
+        decompress: false,
     };
 
     if (json && !decodeBody) {
-        throw new Error('If parseBody is set to true the decodeBody must be also true.');
+        throw new Error('If json is set to true the decodeBody must be also true.');
```

Comment:
BTW it's always better to clarify what is a parameter name in the text. For example:

`If the "json" parameter is true, "decodeBody" must be also true.`